src/app/src/main/kotlin/org/imperial/mrc/hint/controllers/LoginController.kt

```diff
@@ -7,6 +7,21 @@ import org.springframework.ui.set
 import org.springframework.web.bind.annotation.GetMapping
 import javax.servlet.http.HttpServletRequest
 import org.imperial.mrc.hint.AppProperties
+import java.util.*
+
+fun ResourceBundle.getUTF8String(key: String): String
+    {
+        return this.getString(key)
+                .toByteArray(Charsets.ISO_8859_1)
+                .toString(Charsets.UTF_8)
+    }
+
+fun getErrorMessageTranslation(key: String, request: HttpServletRequest): String
+{
+    val language = request.getHeader("Accept-Language") ?: "en"
+    val resourceBundle = ResourceBundle.getBundle("ErrorMessageBundle", Locale(language))
+    return resourceBundle.getUTF8String(key)
+}
```

Comment:
A bit of could repetition here. Could be put in a util perhaps but I didn't feel it was worth it.